includes/blocks/course-list/class-sensei-course-list-filter-block.php

```diff
@@ -63,22 +59,17 @@ private function register_block() {
 	 * @return string
 	 */
 	public function render_block( $attributes, $content, WP_Block $block ): string {
-		if ( ! isset( $attributes['type'] ) ) {
+		if ( ! isset( $attributes['types'] ) || ! is_array( $attributes['types'] ) ) {
 			return '';
 		}
 		$content = '';
 
-		switch ( $attributes['type'] ) {
-			case 'categories':
-				$content = $this->category_filter->get_content( $block->context['queryId'] );
-				break;
-			case 'featured':
-				$content = $this->featured_filter->get_content( $block->context['queryId'] );
-				break;
-			case 'activity':
-			default:
-				break;
+		foreach ( $this->filters as $filter ) {
+			if ( in_array( $filter->get_filter_name(), $attributes['types'], true ) ) {
+				$content .= $filter->get_content( $block->context['queryId'] );
+			}
 		}
+
 		$wrapper_attributes = get_block_wrapper_attributes( [ 'class' => 'wp-sensei-course-list-block-filter' ] );
```

Comment:
I think we can get rid of this class and use `wp-block-sensei-lms-course-list-filter` instead.

Reply:
Thanks, changed relevant parts to use the automatically added class [here](https://github.com/Automattic/sensei/pull/5578/commits/b4337eb0576cc4ae870f6dfaa5fbf67870c88f1e)